payloads/Makefile.mk: add new option to skip auto flash

wiktormowinski opened the pull request:
solution to: https://github.com/Dasharo/dasharo-issues/issues/1402

edk2 part: https://github.com/Dasharo/edk2/pull/264

wiktormowinski committed:
payloads/edk2: add an option to control iPXE boot on FUM

Upstream-Status: Inappropriate [Dasharo downstream]
Signed-off-by: Wiktor Mowinski <wiktor.mowinski@3mdeb.com>

SergiiDmytruk commented on payloads/external/edk2/Kconfig.dasharo:
```diff
@@ -413,4 +413,11 @@ config EDK2_GRAPHICAL_CAPSULE_PROGRESS
+	bool "Don't launch the auto-update sequence in Firmware Update Mode"
+	default n
+	help
+	  A way to disable all flash locks on and use the platform manually, the default
```
What this the intention?
```suggestion
	  A way to disable all flash locks on the next boot and boot the platform as usual, the default
```

wiktormowinski replied:
sure, im not good at naming things: 
https://github.com/Dasharo/coreboot/pull/711/commits/24896daeaceb8dc0dd94056757a365ca9c9d3b5a

SergiiDmytruk reviewed (commented):
How about a name like `FUM_AUTO_IPXE_BOOT`?  EDK doesn't know anything about DTS, it's baked into iPXE image by coreboot which then supplies it to EDK as part of the build.  Also it's better to avoid negation in names as they lead to double negations in conditionals.

SergiiDmytruk commented:
And one more thing: the new option should be utilized in `configs/config.protectli_vp66xx`.

wiktormowinski committed:
configs/config.protectli_vp66xx: disable new iPXE boot option

DTS doesn't support Protectli platforms, so DTS should not be launched
there.

Upstream-Status: Inappropriate [Dasharo downstream]
Signed-off-by: Wiktor Mowinski <wiktor.mowinski@3mdeb.com>

SergiiDmytruk committed:
payloads/edk2: bump to get PcdFumAutoIpxeBoot

Upstream-Status: Inappropriate [Dasharo downstream]
Signed-off-by: Sergii Dmytruk <sergii.dmytruk@3mdeb.com>

SergiiDmytruk commented:
Changing the name inverted meaning of the option and `config` was missing `EDK2` prefix, but I corrected that while testing the functionality.

Also note that you weren't using latest `dasharo` branch locally.  EDK's revision is no longer specified in `configs`, see https://github.com/Dasharo/coreboot/pull/711/commits/6b6440ac4fb86cbe1a14b9d33e689a2b27d456df for the new way of doing it.
